state-synchronizer/src/state_synchronizer.rs

```diff
@@ -297,7 +297,7 @@ impl StateSyncClient {
 
     /// Returns information about the state sync internal state. This should only
     /// be used by tests.
-    #[cfg(test)]
+    // TODO(joshlind): remove this once unit tests are added!
```

Comment:
Btw, I don't think you need to add these kinds of comments, since you're going around the next commit to fix it :)